src/sources/dnstap/mod.rs

```diff
@@ -195,8 +203,20 @@ impl FrameHandler for DnstapFrameHandler {
 
         let mut log_event = LogEvent::default();
 
+        self.log_namespace.insert_vector_metadata(
+            &mut log_event,
+            self.timestamp_key().as_str(),
+            "ingest_timestamp",
+            chrono::Utc::now(),
+        );
+
         if let Some(host) = received_from {
-            log_event.insert(self.host_key().as_str(), host);
+            self.log_namespace.insert_vector_metadata(
```

Comment:
This would be considered `source` metadata (not `vector` metadata) since it's specific to this source (not every source will have this info).

Only the `ingest_timestamp` and `source_type` today are vector metadata. (I'm not seeing `source_type` yet, so that should be added)